agent/lib/src/commands/ci.dart

```diff
@@ -201,6 +204,17 @@ class ContinuousIntegrationCommand extends Command {
     }
   }
 
+  Future<Null> _screensOn() async {
+    try {
+      for (Device device in await devices.discoverDevices()) {
+        await device.wakeUp();
```

Comment:
We don't have to fix it right now, but the fact that we just toggle without detecting the current screen on state seems a bit fragile in the long run.

Reply:
Oh sorry there is a _getWakefulness currently. Never mind.

Reply:
adb.wakeUp does that validation: https://github.com/flutter/cocoon/blob/master/agent/lib/src/adb.dart#L188